src/mixins/instanceMixins/JOIous.js

```diff
@@ -55,9 +64,8 @@ const JOIous = ReceivingClass => class extends ReceivingClass {
     return this.sortAndSerialize()
   }
 
-  // deeper print outs while running on Node
   toString() {
-    return util.inspect(this.toJSON(), { compact: false, depth: 4 })
+    return util.inspect(this, { compact: false, depth: 4 })
   }
 
   // deeper print outs while running on Node
```

Comment:
Should the deeper printout have a higher depth than 4 ?

Reply:
good question and 👁️ ! - I actually chose 4 off the cuff assuming it would get us all the way down to data streams from an FCTGraph printout and no further. Turns out it doesn't (but 6 does!).

`{ fctGraph: { fcts: { slots: { dataStreams } }` 

I have updated it (and also fixed unfound mistake in which datastreams weren't being serialized).